ui/src/views/CubeProject.vue

```diff
@@ -12,6 +12,14 @@
             <a :href="href" @click="navigate">Cube Designer</a>
           </li>
         </router-link>
+        <router-link :to="{ name: 'CubeProjectEdit' }" v-slot="{ href, isActive, navigate }" v-if="project.actions.delete || project.actions.edit">
```

Comment:
Do we have edit actions or will we have?

Reply:
We don't have it yet ;)